src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -125,7 +113,7 @@ const ColonyHome = ({
   intl: { formatMessage },
 }: Props) => {
   const [filterOption, setFilterOption] = useState(TasksFilterOptions.ALL_OPEN);
-  const [filteredDomainId, setFilteredDomainId] = useState(0);
+  const [filteredDomainId, setFilteredDomainId] = useState(1);
```

Comment:
Should it be all or root selected by default? Was there a product decision?

Reply:
But isn't `root` being selected show all tasks in all it's child domains? Meaning all tasks are displayed regardless ?

Reply:
So actually selecting domain `1` will show only tasks from the root domain, and not children. You have to select `0` to show all tasks.